examples/point_transformer_classification.py

```diff
@@ -0,0 +1,213 @@
+        self.transition_down = torch.nn.ModuleList()
+
+        for i in range(len(dim_model) - 1):
+            # Add Transition Down block followed by a Transformer block
+            self.transition_down.append(
+                TransitionDown(in_channels=dim_model[i],
+                               out_channels=dim_model[i + 1],
+                               k=self.k)
+            )
+
+            self.transformers_down.append(
+                TransformerBlock(in_channels=dim_model[i + 1],
+                                 out_channels=dim_model[i + 1])
+            )
+
+        # class score computation
+        self.lin = Seq(Lin(dim_model[-1], 64),
+                       ReLU(),
+                       Lin(64, 64),
+                       ReLU(),
+                       Lin(64, NUM_CLASSES))
+
+    def forward(self, data):
+        x, pos, batch = data.x, data.pos, data.batch
+
+        # add dummy features in case there is none
+        if x is None:
+            x = torch.ones((pos.shape[0], 1)).to(pos.get_device())
+
+        # first block
+        x = self.mlp_input(x)
+        edge_index = knn_graph(pos, k=self.k, batch=batch)
+        x = self.transformer_input(x, pos, edge_index)
+
+        # backbone
+        for i in range(len(self.transformers_down)):
+            x, pos, batch = self.transition_down[i](x, pos, batch=batch)
+
+            edge_index = knn_graph(pos, k=self.k, batch=batch)
+            x = self.transformers_down[i](x, pos, edge_index)
+
+        # GlobalAveragePooling
+        x = global_mean_pool(x, batch)
+
+        # Class score
+        out = self.lin(x)
+
+        return F.log_softmax(out, dim=-1)
+
+
+def train():
+    model.train()
+
+    total_loss = 0
+    for data in train_loader:
+        data = data.to(device)
+        optimizer.zero_grad()
+        out = model(data)
+        loss = F.nll_loss(out, data.y)
+        loss.backward()
+        total_loss += loss.item() * data.num_graphs
+        optimizer.step()
+    return total_loss / len(train_dataset)
+
+
+def test(loader):
+    model.eval()
+
+    correct = 0
+    for data in loader:
+        data = data.to(device)
+        with torch.no_grad():
+            pred = model(data).max(dim=1)[1]
+        correct += pred.eq(data.y).sum().item()
+    return correct / len(loader.dataset)
+
+
+if __name__ == '__main__':
+
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    model = Net(train_dataset.num_classes, 1, dim_model=[
+                32, 64, 128, 256, 512], k=16).to(device)
+    optimizer = torch.optim.Adam(model.parameters(), lr=0.001)
+    scheduler = torch.optim.lr_scheduler.StepLR(optimizer,
+                                                step_size=20,
+                                                gamma=0.5)
+
+    for epoch in range(1, 201):
+        loss = train()
+        test_acc = test(test_loader)
+        print('Epoch {:03d}, Loss: {:.4f}, Test: {:.4f}'.format(
```

Comment:
Prefer `f`-string syntax:
```suggestion
        print(f'Epoch {epoch:03d}, Loss: {loss:.4f}, Test: {test_acc:.4f}')
```